src/index.html

```diff
@@ -6,9 +6,51 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
+    <!-- <link rel="stylesheet" href="./normalize.css"> -->
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home"
+      class="logo">
+        <img src="./images/logo.png"
+        alt="MOYO" class="logo__img">
+      </a>
```

Comment:
```suggestion
      <a href="#home" class="logo">
        <img src="./images/logo.png" alt="MOYO" class="logo__img">
      </a>
```

Reply:
Only 2 attributes per line are permittedlinthtml(attr-new-line)

Reply:
<a href="#home" class="logo">
        <img
        src="./images/logo.png"
        alt="MOYO"
        class="logo__img"
        >
        Сould I do this change, because my VS code wrote me mistake "Only 2 attributes per line are permitted lint html (attr-new-line)"?